charts/prometheus-blackbox-exporter/README.md

```diff
@@ -48,6 +48,10 @@ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### To 8.5.0
+
+- Support Google Managed Prometheus by creating `PodMonitoring` custom resource (CR) when setting extra targets
+
```

Comment:
changes are only needed for major versions

```suggestion
```

Reply:
removed as suggestion